### PR TITLE
Fix typo in clang-format directive and reformat end of flowgraph.cpp

### PR DESCRIPTION
**Pull request by AndyAyersMS**
Last 4K or so lines of flowgraph.cpp were not being formatted because
the clang-format on directive had a typo.

Fix the typo and reformat the latter part of the file.

### CODE REVIEW

**Comment by AndyAyersMS** [comment]
cc @dotnet/jit-contrib

**Comment by AndyAyersMS** [comment]
There are some ugly reformats in here which might warrant more format off/ons. 

Look around new line #s 19678 and 20260....

**Comment by AndyAyersMS** [comment]
FYI no other bad off/on pairs I could see.

File: src/jit/flowgraph.cpp [hunk 1]
```diff
@@ -20245,14 +20257,14 @@ PRED_OK:;
...
-#endif  // !JIT32_GCENCODER
-        noway_assert(compThisArgAddrExposedOK &&                  //     should never expose the address of arg 0 or
-                     !lvaTable[info.compThisArg].lvArgWrite &&    //     write to arg 0.
-                     (                                                  //   In addition,
-                        lvaArg0Var == info.compThisArg ||               //     lvArg0Var should remain 0 if arg0 is not written to or address-exposed.
-                        lvaArg0Var != info.compThisArg &&
-                        (lvaTable[lvaArg0Var].lvAddrExposed || lvaTable[lvaArg0Var].lvArgWrite || copiedForGenericsCtxt)
-                     ));
+#endif                                                         // !JIT32_GCENCODER
+        noway_assert(compThisArgAddrExposedOK &&               //     should never expose the address of arg 0 or
```
Comment (adiaaida) [inline src/jit/flowgraph.cpp]: Can we move the comment on this assert to be above it? The way it is currently written is super confusing

File: src/jit/flowgraph.cpp [hunk 2]
```diff
@@ -20308,59 +20320,62 @@ void                Compiler::fgDebugCheckFlags(GenTreePtr tree)
         // Is the left child of "tree" a GT_ASG?,
         if (op1 && op1->gtOper == GT_ASG)
         {
-            assert(tree->gtType == TYP_VOID ||  // If parent is a TYP_VOID, we don't no need to propagate TYP_INT up. We are fine.
-                   tree->gtOper == GT_COMMA);   // (or) If GT_ASG is the left child of a GT_COMMA, the type of the GT_COMMA node will
-        }                                       // be determined by its right child. So we don't need to propagate TYP_INT up either. We are fine.
+            assert(tree->gtType ==
```
Comment (adiaaida) [inline src/jit/flowgraph.cpp]: Perhaps move this comment too.

File: src/jit/flowgraph.cpp [hunk 3]
```diff
@@ -22150,23 +22123,18 @@ GenTreePtr Compiler::fgInlinePrependStatements(InlineInfo* inlineInfo)
                         noway_assert(structHnd != NO_CLASS_HANDLE);
                     }
 
-                    // Unsafe value cls check is not needed for argTmpNum here since in-linee compiler instance would have
+                    // Unsafe value cls check is not needed for argTmpNum here since in-linee compiler instance would
```
Comment (adiaaida) [inline src/jit/flowgraph.cpp]: Reflow comment

File: src/jit/flowgraph.cpp [hunk 4]
```diff
@@ -19640,47 +19655,39 @@ void                Compiler::fgDispBasicBlocks(BasicBlock*  firstBlock,
...
-        );
-    printf("------%*s------------------------------------%*s-----------------------%*s----------------------------------------\n",
-        padWidth, "------------",
-        ibcColWidth, "------------",
-        maxBlockNumWidth, "----");
-
-    for (block = firstBlock;
-         block;
-         block = block->bbNext)
+    printf("------%*s------------------------------------%*s-----------------------%*s---------------------------------"
```
Comment (adiaaida) [inline src/jit/flowgraph.cpp]: We potentially want to surround these printfs with clang-format off and clang-format on

**Comment by adiaaida** [comment]
Modulo those comments, looks good.

**Comment by AndyAyersMS** [comment]
Going to merge this. If anyone else spots formatting that they don't like, let me know.

**Pull request merged**